cypress.config.js

```diff
@@ -8,11 +8,18 @@ module.exports = defineConfig({
       on('task', {
         generateUser() {
           const email = faker.internet.email();
-          const randomNumber = Math.floor(Math.random(1000) * 1000);
+          const randomNumber = Math.floor(Math.random(1000) * 1000) + 1;
```

Comment:
It's not clear why you're generating a random number up to 1000. If this is to ensure unique usernames, consider using a more reliable method, such as a UUID or a timestamp.